app/js/controllers/main.js

```diff
@@ -20,7 +24,8 @@ export default class MainController extends BaseController {
 
     const mountNode = document.querySelector('.app-view-container');
     const speechController = new SpeechController();
-    const server = new Server();
+    const settings = new Settings();
+    const server = new Server({ settings });
```

Comment:
actually in the current version we already have one version only -- server/index.js creates one only if it has no object passed as parameter. I could remove this code in server/index.js to throw instead, as you wish.